turbo.json

```
@@ -20,17 +20,21 @@
       "outputs": ["dist/**"],
       "dependsOn": ["^build", "compile"]
     },
+    "build:docker": {
+      "inputs": ["Dockerfile", "./"],
```

```suggestion
      "inputs": ["**/src/**", "**/Cargo.toml"],
```

We should be more explicit.

> And there isn't a file called "meta/Dockerfile", so putting `Dockerfile` here won't do anything. Which is also somewhat confusing.